opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -249,19 +249,40 @@ namespace Opm {
         return m_minpvValue;
     }
 
+    /// static helper function checking that _one_ coordinate differs with 1
+    bool ijk_are_adj(int i1, int j1, int k1, int i2, int j2, int k2) {
```

Comment:
static or not, so far the rest of opm-parser uses camelCase (instead_of_underscores) as its naming scheme for methods and classes. is the name `ijk_are_adj()' intentional?